src/wsi/sdl3/wsi_platform_sdl3_funcs.h

```diff
@@ -0,0 +1,2 @@
+// TODO SDL3
```

Comment:
Once the build works again, fill this in to remove the linker requirement. See wsi_platform_sdl2_funcs for an example.